tests/integration/rest.integration.test.ts

```diff
@@ -14,7 +14,7 @@ describe('HiFi API REST Calls', () => {
             nonAdminToken = await generateJWT(TOKEN_GEN_TYPES.NON_ADMIN_ID_APP1_SPACE1_SIGNED);
         } catch (err) {
             console.error("Unable to create non admin token for testing REST calls. ERR: ", err);
-            return;
+            process.exit();
```

Comment:
is this something that needs to happen after every test or just this one?

Reply:
This only needs to happen here if we can't set up for the tests by creating the admin token. I've actually tried adding it after every test to fix the 'worker process has failed to exit...' bug but then the test suite stops completely after the first test. I can only call this once per file and then the script stops.

Reply:
the first sentence makes sense to me

the latter part of your explanation also makes sense - if you call `process.exit()` at any time, that entire test suite will stop at the time that function is called.